features/FxA-89-devices-view/README.md

```diff
@@ -411,6 +409,17 @@ Acceptance criteria:
       by accessing the devices view on a desktop browser,
       and then go to the sync preferences on the mobile device,
       I immediately see that the device is disconnected.
+* [ ] - Moved from Phase 2: When viewing my device list
```

Comment:
@rfk this is a weird case because MOST users are sync users, so they should have at least one in there.